### PR TITLE
Fixed errors, removed UserProfile.js functionality temporarily, Added…

### PR DESCRIPTION
**Pull request by Nikman800**
… menu button with settings page and myprofile page access.

Okay, should fix the main issues.

### CODE REVIEW

**Comment by Nikman800** [comment]
Only issue is I am unsure how to link backend at the moment. Also how to link with azure.

**Review by jmdebarro — APPROVED** [review]
Looks great, nice job fixing merge conflicts. Thanks for commenting out the stuff we won't be able to finish

File: packages/react-frontend/src/components/Menu_Button.png
Comment (Alecs-Games) [inline packages/react-frontend/src/components/Menu_Button.png:1]: Rotating this image 90 degrees to make the bars vertical would be more consistent with most UI designs

File: packages/react-frontend/src/components/MyProfile.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import MainComponent from "./Item";
+import styles from "./MyProfile.module.css"; // Import the CSS module
+
+const items = [
```
Comment (Alecs-Games) [inline packages/react-frontend/src/components/MyProfile.js:5]: These can be deleted once connected to backend. Should be a very quick drop-in replacement

File: packages/react-frontend/src/components/Settings.js [hunk 2]
```diff
@@ -3,9 +3,35 @@ import { Link } from "react-router-dom";
 import style from "./settings.module.css"
 
 function Settings() {
-    return <div className={style.container}>
-        <button className={style.button}><Link to="/login">Log In</Link></button>
-        <button className={style.button}><Link to="/signup">Sign Up</Link></button>
+  return (
+    <div className={style.container}>
+      <div className={style.profile}>
+        <div className={style.profileBubble}></div>{" "}
```
Comment (Alecs-Games) [inline packages/react-frontend/src/components/Settings.js:9]: Do users have a profile image at this point?

Comment (ariannawilde3) [reply packages/react-frontend/src/components/Settings.js:9]: I do not believe users have a profile image

**Comment by ariannawilde3** [comment]
Nice Job with the menu button, does it have the whole page done as well? Or just the button showing? If the page is there I can connect it to the backend.

**Pull request merged**